packages/fxa-payments-server/src/lib/test-utils.tsx

```diff
@@ -179,7 +179,6 @@ export const elementChangeResponse = ({
 });
```

Comment:
I think I know why you're seeing connection refused errors - we need to set up mock config for the new singleton API client in test-utils. Otherwise, the API client will try to make real network requests instead of hitting the mock handlers in tests. These changes get most of the tests passing for me:
```diff
diff --git a/packages/fxa-payments-server/src/lib/test-utils.tsx b/packages/fxa-payments-server/src/lib/test-utils.tsx
index 68c2676c5..0d2ef5f8d 100644
--- a/packages/fxa-payments-server/src/lib/test-utils.tsx
+++ b/packages/fxa-payments-server/src/lib/test-utils.tsx
@@ -2,6 +2,7 @@ import React, { useContext, ReactNode } from 'react';
 import { Provider as ReduxProvider } from 'react-redux';
 import { AppContext, AppContextType } from '../../src/lib/AppContext';
 import { config, updateConfig } from '../../src/lib/config';
+import { updateAPIClientToken, updateAPIClientConfig } from '../../src/lib/apiClient';
 import ScreenInfo from '../../src/lib/screen-info';
 import { ReactStripeElements } from 'react-stripe-elements';
 import nock from 'nock';
@@ -17,6 +18,8 @@ declare global {
   }
 }
 
+export const MOCK_ACCESS_TOKEN = '8675309jenny';
+
 export const wait = (delay: number) =>
   new Promise(
     resolve => setTimeout(resolve, delay)
@@ -79,6 +82,8 @@ export function mockOptionsResponses(baseUrl: string) {
 
 export const setupMockConfig = (config?: typeof mockConfig) => {
   updateConfig(config || mockConfig);
+  updateAPIClientConfig(config || mockConfig);
+  updateAPIClientToken(MOCK_ACCESS_TOKEN);
 };
 
 // Minimal mock for react-stripe-elements that lets us trigger onChange
```